openmetadata-ui/src/main/resources/ui/src/components/EntityLineage/EntityLineage.component.tsx

```diff
@@ -213,14 +219,33 @@ const EntityLineageComponent: FunctionComponent<EntityLineageProp> = ({
   });
 
   const params = useParams<Record<string, string>>();
+  const queryParams = new URLSearchParams(location.search);
+  const isFullScreen = queryParams.get('fullscreen') === 'true';
   const entityFQN =
     params[getParamByEntityType(entityType)] ?? params['entityFQN'];
   const history = useHistory();
 
   const onFullScreenClick = useCallback(() => {
-    history.push(getLineageViewPath(entityType, entityFQN));
+    history.push({
+      search: Qs.stringify({ fullscreen: true }),
+    });
   }, [entityType, entityFQN]);
 
+  const breadcrumbs = useMemo(
+    () =>
+      entity
+        ? [
+            ...getEntityBreadcrumbs(entity, entityType as EntityType),
```

Comment:
I recommend changing `entityType` from string type to `EntityType` so we don't have to typecast it.

Reply:
Yup addressed.